hosting/single/Dockerfile

```diff
@@ -12,13 +12,15 @@ RUN chmod +x /cleanup.sh
 WORKDIR /app
 ADD packages/server .
 COPY yarn.lock .
-RUN yarn install --production=true && /cleanup.sh
+RUN yarn install --production=true
```

Comment:
Just splitting it for more verbosity on the build. This was failing and it was hard to see if it was because the yarn install or the cleanup. It should have no impact on the performance, as it was already 2 different steps